src/Practice4_QuanLyHocSinh/School.java

```diff
@@ -0,0 +1,18 @@
+package Practice4_QuanLyHocSinh;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class School {
+	
+	private List<Student> studentList = new ArrayList<Student>();
```

Comment:
```suggestion
	private List<Student> students = new ArrayList<Student>();
```
đổi tên beiens cái nhìn pro hơn hẳng nè :v